include/alpaka/core/ConcurrentExecPool.hpp

```diff
@@ -73,7 +73,7 @@ namespace alpaka
                 {
                     std::lock_guard<std::mutex> lk(m_Mutex);
 
-                    std::queue<T>::push(std::forward<T>(t));
+                    std::queue<T>::push(t);
```

Comment:
But `t` is a reference to a function local shared pointer of a thread different to the worker thread. Therefore it is not allowed to hold a reference.